packages/grid/x-data-grid-generator/src/commodities.columns.tsx

```diff
@@ -245,6 +245,10 @@ export const getCommodityColumns = (editable = false): GridColDefGenerator[] =>
     groupingValueGetter: (params) => params.value.code,
     type: 'singleSelect',
     valueOptions: COUNTRY_ISO_OPTIONS,
+    sortComparator: (v1, v2) =>
```

Comment:
Could you re-use the built-in comparators lilke in [this new demo](https://github.com/mui-org/material-ui-x/pull/3564/files#diff-ed5c7d4fcb50eaed3b957b22690f68490290152b9f85c3c7accad98972316f37R19) ?

Reply:
Sure, but we'll need to merge https://github.com/mui-org/material-ui-x/pull/3564 first, since they're not exported yet.

Reply:
Oh you are right, this should come quickly :+1:

Reply:
Merged

Reply:
Done!